CITATION.cff

```diff
@@ -0,0 +1,25 @@
+cff-version: 1.0.0
+message: "If you use this software, please cite it as below."
+authors:
```

Comment:
I think We'll want everyone who's a contributor listed here. We can fill the ORCID later. So it should also include:
- Adam Coogan
- Andreas Filipp
- Charles Wilson
- Misha Yantovski-Barth
- Ronan Legin